src/applications/new-28-1900/containers/ConfirmationPage.jsx

```diff
@@ -24,57 +41,107 @@ export const ConfirmationPage = () => {
+              <h3 className="vads-u-font-size--h4 vads-u-margin-top--2 vads-u-margin-bottom--0p5">
+                Date submitted
+              </h3>
+              {submitDateHumanFriendly}
+            </>
+          ) : null}
+          <h3 className="vads-u-font-size--h4 vads-u-margin-top--2 vads-u-margin-bottom--0p5">
+            Confirmation for your records
+          </h3>
+          You can print this confirmation page for your records
+          <br />
+          <div className="vads-u-margin-top--2">
+            <va-button onClick={window.print} text="Print this page" />
+          </div>
+        </va-summary-box>
+      </div>
+      <h2>What to expect next</h2>
+      <p>
+        We’ll review your application to determine if you’re eligible for an
+        initial evaluation with a Vocational Rehabilitation Counselor (VRC). If
+        we determine you’re eligible for an initial evaluation, we’ll send you
+        an appointment letter with the date and time when a VRC will meet with
+        you.
+      </p>
+      <p>
+        During the initial evaluation, the VRC will gather information to
+        determine if you’re entitled to receive VR&E benefits.
+      </p>
+      <h2>How to contact us if you have questions</h2>
+      <p>
+        You can ask us about your application online.{' '}
+        <a href="https://ask.va.gov/" target="_blank" rel="noreferrer">
+          Contact us online through Ask VA (opens in new tab)
+        </a>
+      </p>
+      <p>
+        Or call us at <va-telephone contact={CONTACTS.VA_BENEFITS} /> (TTY:{' '}
+        <va-telephone contact={CONTACTS[711]} />
+        ). We’re here Monday through Friday, 8:00 am to 8:00 pm ET.
+      </p>
+      <p>
+        <strong>If you don’t receive a decision letter,</strong> don’t apply
+        again. Contact us online or call us instead.
+      </p>
+
+      <a className="vads-c-action-link--green vads-u-margin-top--3" href="/">
         Go back to VA.gov
       </a>
-
-      {/* <div className="help-footer-box">
-        <h2 className="help-heading">Need help?</h2>
-        <GetFormHelp />
-      </div> */}
+      <div className="help-footer-box vads-u-margin-top--6">
+        <h2 className="help-heading vads-u-font-size--h3">Need help?</h2>
+        <p className="help-talk">
+          <strong>If you have trouble using this online form,</strong> call us
+          at <va-telephone contact="8006982411" /> (
+          <va-telephone contact="711" tty />
+          ). We&rsquo;re here 24/7.
+        </p>
+        <p className="help-talk">
+          <strong>
+            If you need help gathering your information or filling out your
+            form,
+          </strong>{' '}
+          contact a local Veterans Service Organization (VSO).
+        </p>
+        <va-link
+          href="https://www.va.gov/get-help-from-accredited-representative/"
+          text="Get help from a VA accredited representative or VSO"
+        />
+      </div>
```

Comment:
The design differed in many ways from the ConfirmationContainer, so I filled in our confirmation page without it.